cmd/osmosisd/cmd/balances_from_state_export.go

```diff
@@ -363,7 +346,6 @@ Example:
 				//ASSUMING THAT "addr" IS THE KEY
 				delete(snapshotAccs, addr)
```

Comment:
@ValarDragon @mattverse Is this a correct/valid way to remove an account from the snapshotAccs map? Since we don't have a proper testing suite for this file I want to make sure I'm not using a function that breaks something